### PR TITLE
rgb::Image: rename alpha_premultiplied to premultiply_alpha

### PR DESCRIPTION
**Pull request by vigneshvg**
Makes it clear that it is an input parameter.

### CODE REVIEW

File: src/utils/raw.rs [hunk 1]
```diff
@@ -41,7 +41,7 @@ impl RawWriter {
             rgb.format = rgb::Format::Rgba;
             rgb.depth = 16;
             //rgb.depth = 8;
-            rgb.alpha_premultiplied = true;
+            rgb.premultiply_alpha = true;
```
Comment (jzern) [inline src/utils/raw.rs:44]: leftover code at the line above this?

File: src/reformat/rgb.rs [hunk 2]
```diff
@@ -91,7 +91,7 @@ pub struct Image {
     pub format: Format,
     pub chroma_upsampling: ChromaUpsampling,
     pub chroma_downsampling: ChromaDownsampling,
-    pub alpha_premultiplied: bool,
+    pub premultiply_alpha: bool,
```
Comment (jzern) [inline src/reformat/rgb.rs:94]: Just so I understand, is this a command to premultiply the alpha or the state of the image?

**Review by jzern — APPROVED** [review]

File: src/utils/raw.rs (hunk 1 above)
Comment (vigneshvg) [reply src/utils/raw.rs:44]: yeah this whole file is sort of a place holder that's not really used. i change it periodically for testing.

File: src/reformat/rgb.rs (hunk 2 above)
Comment (vigneshvg) [reply src/reformat/rgb.rs:94]: this is an input parameter that tells the conversion functions whether or not to premultiply the alpha.

once the conversion function is called, this also reflects the state of the image. ideally we should be splitting the parameters and the image state. it's on my list, will do it later.

**Pull request merged**